src/functions.ts

```diff
@@ -46,6 +46,7 @@ export const createFunctions = async (options: Options, manifest: any) => {
     const result = await Deno.emit(fnFilePath, {
       bundle: "module",
       check: false,
+      importMapPath: `${options.workingDirectory}/import_map.json`,
```

Comment:
If we don't want to account for the user provided `source` arg because the CLI requires `import_map.json` at the project root, we can do this instead

```suggestion
      importMapPath: path.join(Deno.cwd(), "import_map.json"),
```